test/SafeProtocolManager.spec.ts

```diff
@@ -94,6 +94,21 @@ describe("SafeProtocolManager", async () => {
                     .withArgs(pluginAddress, 0, 0);
             });
 
+            it("Should not allow a Safe to enable plugin that does not support ERC165", async () => {
```

Comment:
should it also test an eoa case?

Reply:
eoa enabling a plugin or if plugin is an EOA?

Reply:
in such case tx would fail anyways. i will add a test though

Reply:
https://github.com/5afe/safe-core-protocol/pull/74/commits/f44918a98299a65ff4ac35c7ef8c07e7893585cc